playgrounds/astro/src/pages/_layout.astro

```diff
@@ -0,0 +1,25 @@
+---
+import { ThemeProvider } from '@itwin/itwinui-react';
+---
+
+<html lang='en'>
+  <head>
+    <meta charset='utf-8' />
+    <meta name='viewport' content='width=device-width' />
+    <title>Astro</title>
+  </head>
+  <ThemeProvider as='body' theme='dark'>
+    <slot />
+  </ThemeProvider>
+
+  <style is:global>
```

Comment:
What is the `is:global` for? Is it an astro specific thing?

Reply:
yes https://docs.astro.build/en/reference/directives-reference/#isglobal